openbas-api/src/main/java/io/openbas/rest/inject/service/InjectStatusService.java

```diff
@@ -39,13 +37,30 @@ public Inject updateInjectStatus(String injectId, InjectUpdateStatusInput input)
     // build status
     InjectStatus injectStatus = new InjectStatus();
     injectStatus.setInject(inject);
-    injectStatus.setTrackingSentDate(now());
     injectStatus.setName(ExecutionStatus.valueOf(input.getStatus()));
     // Save status for inject
     inject.setStatus(injectStatus);
     return injectRepository.save(inject);
   }
 
+  public void addStartImplantExecutionTraceByInject(
+      String injectId, String agentId, String message) {
+    Inject inject = injectRepository.findById(injectId).orElseThrow();
```

Comment:
Would it be possible to not fetch the whole inject but simply an injectstatus instead ? It doesn't look like we need data from the inject it self and it should be quicker.